apps/etterlatte-behandling/src/test/kotlin/OmberegningIntegrationTest.kt

```diff
@@ -0,0 +1,119 @@
+package no.nav.etterlatte
+
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import io.ktor.client.call.body
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.request.get
+import io.ktor.client.request.header
+import io.ktor.client.request.post
+import io.ktor.client.request.setBody
+import io.ktor.http.ContentType
+import io.ktor.http.HttpHeaders
+import io.ktor.http.HttpStatusCode
+import io.ktor.serialization.jackson.jackson
+import io.ktor.server.testing.testApplication
+import no.nav.etterlatte.behandling.BehandlingsBehov
+import no.nav.etterlatte.libs.common.behandling.BehandlingListe
+import no.nav.etterlatte.libs.common.behandling.BehandlingType
+import no.nav.etterlatte.libs.common.behandling.Omberegningshendelse
+import no.nav.etterlatte.libs.common.behandling.Persongalleri
+import no.nav.etterlatte.libs.common.behandling.RevurderingAarsak
+import no.nav.etterlatte.libs.common.behandling.SakType
+import no.nav.etterlatte.sak.Sak
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.*
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class OmberegningIntegrationTest : BehandlingIntegrationTest() {
+
+    @BeforeAll
+    fun start() = startServer()
+
+    @AfterAll
+    fun shutdown() = afterAll()
+
+    @Test
+    fun omberegn() {
```

Comment:
Opplever at integrasjonstestene ofte blir uryddige, så et velbeskrivende funksjonsnavn hadde vært fint for "dokumentasjon"

Reply:
Veldig sant.